### DISCUSSION TITLE
gradle clean build 报错，怎么解决啊

### DISCUSSION THREAD

**Discussion by hujingguang** [Tencent/bk-ci#4580] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/11848843/124097314-0a6c1a00-da8e-11eb-8798-4c71811ec100.png)

**Comment by zanyzhao** [comment, marked as answer]
- 可以补充下本地的编译环境具体信息：比如gradle版本等
- 可以参考我们配置的[GitHub Actions](https://github.com/Tencent/bk-ci/actions/runs/995854326/workflow)流程做参考

**Comment by mikeeastin** [comment]
update  gradle version to  6.8   fixed
